samples/openthread/cli/overlay-logging.conf

```diff
@@ -25,4 +25,5 @@ CONFIG_OPENTHREAD_DEBUG=y
 # Option for configuring log level in OpenThread
 CONFIG_OPENTHREAD_LOG_LEVEL_INFO=y
 
-# Adjust log strdup settings
+# Increase log buffer size for OpenThread
+CONFIG_LOG_BUFFER_SIZE=4048
```

Comment:
Maybe move this configuration to `nrf/subsys/net/openthread/Kconfig.defconfig` instead?

```
config LOG_BUFFER_SIZE
	default 4048 if LOG
```